src/main/resources/com/cloudbees/jenkins/support/SupportAction/index.jelly

```diff
@@ -37,31 +37,11 @@
       <p>
         ${%detail}
       </p>
-      <f:form name="bundle-contents" method="POST" action="download">
-        <j:forEach var="component" items="${it.components}">
-          <f:entry field="component">
-            <div name="components">
-              <j:if test="${component.enabled}">
-                <f:checkbox name="selected" checked="${it.selectedByDefault(component)}"
-                            title="${component.displayName}"/>
-              </j:if>
-              <j:if test="${!component.enabled}">
-                <input type="checkbox"
-                       name="selected"
-                       value="false"
-                       disabled="disabled"/>
-                <label class="attach-previous" style="text-decoration: line-through;"
-                       title="${%permissionPreReqs(component.displayPermissions)}">${component.displayName}
-                </label>
-              </j:if>
-              <input style="display:none" name="name" value="${component.id}"/>
-            </div>
-          </f:entry>
-        </j:forEach>
-        <f:entry>
-          <f:submit value="${%Generate Bundle}"/>
-        </f:entry>
-      </f:form>
+
+      <a href="BundleGenerator">Download Bundles</a>
```

Comment:
Nice to have: internalization support. https://wiki.jenkins-ci.org/display/JENKINS/Internationalization